api/middleware.go

```diff
@@ -48,3 +48,11 @@ func TokenMiddleware(psk string, public map[string]string, h http.Handler) http.
 		h.ServeHTTP(w, r)
 	})
 }
+
+func ParamMiddleware(h http.Handler) http.Handler {
```

Comment:
This isn't used anywhere, was it just for testing?

Reply:
good catch - I was playing with parsing the query in middleware but decided against it.